typings/fengari.d.ts

```diff
@@ -0,0 +1,186 @@
+  export const FENGARI_AUTHORS: string;
+  export const FENGARI_COPYRIGHT: string;
+  export const FENGARI_RELEASE: string;
+  export const FENGARI_VERSION: string;
+  export const FENGARI_VERSION_NUM: number;
+  export const FENGARI_VERSION_MAJOR: string;
+  export const FENGARI_VERSION_RELEASE: string;
+  export const FENGARI_VERSION_MINOR: string;
+
+  export type lua_String = Uint8Array;
+
+  export function to_luastring(s: string): lua_String;
+  export function to_jsstring(s: lua_String): string;
+  export function to_uristring(s: lua_String): string;
+
+  export namespace lua {
+    // tslint:disable:class-name
+    interface lua_State {
+    }
+    interface lua_Integer {
+    }
+
+    const LUA_OK: number;
+    const LUA_ERRRUN: number;
+
+    type lua_Alloc = (...a: any) => any;
+    type lua_Cfunction = (...a: any) => any;
+    type lua_Number = number;
+    type lua_Writer = (L: lua_State, p: any, sz: number, ud: any) => number; //todo
+    type lua_Reader = (L: lua_State, ud: any, size: any) => any; //todo
+    //tslint:enable
+    const LUA_AUTHORS: string;
+    const LUA_COPYRIGHT: string;
+    const LUA_VERSION: string;
+    const LUA_RELEASE: string;
+
+    const LUA_REGISTRYINDEX: number;
+
+    function lua_absindex(L: lua_State, idx: number): number;
+    function lua_arith(L: lua_State, op: number): void;
+    function lua_atnativeerror(L: lua_State, errfunc: lua_Cfunction): lua_Cfunction;
+    function lua_atpanic(L: lua_State, panicf: lua_Cfunction): lua_Cfunction;
+
+    function lua_call(L: lua_State, nargs: number, nresults: number): void;
+//    function lua_callk(L: lua_State, nargs: number, nresults: number): void;
```

Comment:
why is this commented out?

Reply:
Will need to add `lua_KContext` and `lua_KFunction`.